openquake/commonlib/tests/reinsurance_test.py

```diff
@@ -945,6 +980,17 @@ def test_limit_is_negative(self):
                       "float -5000.0 < 0, line 11, line 11",
                       str(ctx.exception))
 
+    def test_limit_is_empty(self):
+        csvfname = general.gettemp(CSV_NP)
+        xmlfname = general.gettemp(
+            XML_NP.format(csvfname).replace(
+                'limit="5000"', 'limit=""'))
+        with self.assertRaises(ValueError) as ctx:
+            reinsurance.parse(xmlfname, policy_idx)
+        self.assertIn("Could not convert limit->positivefloat: "
+                      "Got an empty string, line 11, line 11",
```

Comment:
Why line 11 appears twice?

Reply:
I forgot to point out that it already happens also elsewhere, so we should investigate the reason

Reply:
I investigated further and it is not trivial to solve this duplication of the line number, so I propose to address it as a separate new issue

Reply:
Done